src/lambda/wizards/samDeployWizard.ts

```diff
@@ -220,5 +378,24 @@ async function getTemplateChoices(
         result.push(new SamTemplateQuickPickItem(uri))
     }
 
-    return result
+    // Identify which labels are non-unique
```

Comment:
This whole method feels messy to me. What do you think about something like this?

```typescript
const uris = onDetectLocalTemplates({ workspaceUris: workspaceFolders })
const labels = new Set<string>(uris.map(uri => getLabel(uri)))

return uris
    .map(uri => new SamTemplateQuickPickItem({
        uri,
        showWorkspaceInDescription: labels.has(getLabel(uri))
    }))
    .sort((a, b) => a.compareTo(b))
```